packages/roosterjs-editor-dom/lib/table/VTable.ts

```diff
@@ -120,6 +145,237 @@ export default class VTable {
         );
     }
 
+    /**
+     * Apply the given table format to this virtual table
+     * @param format Table format to apply
+     */
+    applyFormatAndStyle(format: TableFormat) {
+        if (!this.table) {
+            return;
+        }
+        format = { ...DEFAULT_FORMAT, ...format };

```

Comment:
Here we should use current format of the table instead of default format. Otherwise if I pass in a partial format, all other properties will be replaced with default value